src/components/RadioGroup/RadioGroupItem.tsx

```diff
@@ -24,11 +23,12 @@ export interface IRadioGroupItemProps {
   checkedChanged?: ComponentEventHandler<IRadioGroupItemProps>
   children?: ReactChildren
   className?: string
+  content?: React.ReactNode
   defaultChecked?: boolean
   disabled?: boolean
   icon?: ItemShorthand
-  label?: string
   name?: string
+  shouldFocus?: boolean
```

Comment:
not sure whether we would like to expose it as a prop - most probably this aspect will be handled by accessibility behavior. Could we omit it for this PR, or there is specific urgent need for this functionality to be introduced?

Reply:
this is solving the bug I was telling you about.

#### Repro:
- create a `RadioGroup` with a `RadioGroupItem` with input (as in the examples of this PR);
- click on input;

#### Expected:
- item is checked and focus stays on input (see `Teams` `Add people` dialog in a group chat)

#### Actual:
- the radio item steals the focus;

This is the reason we're using this `shouldFocus` prop in order to set focus only when it's coming from keyboard; 

Unfortunately accessibility functionality spilled in our component logic and this became necessary now to do it like this. Alternative (and probably better approach) would be to rethink the whole accessibility part

What do you think?
@kuzhelov @mnajdova @jurokapsiar

Reply:
fyi @smykhailov

Reply:
adding this prop is critical to enable keyboard navigation for this component ASAP; I created an issue #306  for implementing it properly once @smykhailov 's work in #256 is merged

fyi @mnajdova @kuzhelov @jurokapsiar 

It'd be great to approve this ASAP so we can unblock customers

Reply:
I agree with @Bugaa92 on this. We can come back and refactor this, otherwise this basic functionality will be blocked for at least a week from now unfortunately.